extension/scripts/features/bank-investment/ttBankInvestment.js

```diff
@@ -0,0 +1,210 @@
+"use strict";
+
+(async () => {
+	featureManager.registerFeature(
+		"Bank Investment",
+		"bank",
+		() => settings.pages.bank.investment,
+		null,
+		initialize,
+		teardown,
+		{
+			storage: ["settings.pages.bank.investment"],
+		},
+		null
+	);
+
+	const DAYS_IN_YEAR = 365;
+	const BALANCE = 2_000_000_000;
+	const PERIOD_TYPE = {
+		OneWeek: "1w",
```

Comment:
This naming is triggering me though. It's basically a constant, so it should be UPPERCASE instead of UpperCamelCase.